### ISSUE TITLE
Add sialic acid xref to GlyGen

### ISSUE THREAD

**Issue by kmartinez834** [glygener/glygen-issues#843]
NCBI have now populated independent pages for each of the sialic acids. Please see [https://www.ncbi.nlm.nih.gov/glycans/sialic.html](https://urldefense.com/v3/__https://www.ncbi.nlm.nih.gov/glycans/sialic.html__;!!Mih3wA!HSrsBsCuZsG7_Pyo42NlLz0omlaySnhN62kVOnugEkLkMLu8M2ivGWkxwOPdjSzYZE4bl5J_GPQj8RVXoe6bDT4$) where these pagelinks can be found. 

CID and SID (compound and structure ID) have been assigned to these entities and thus the information curated by the group can now find its way to pubchem partners and the semantic web. Similar work is currently underway for the SNFG monosaccharides, and Evan’s team has also initiated an SNFG reference collection at: [https://pubchem.ncbi.nlm.nih.gov/source/11743](https://urldefense.com/v3/__https://pubchem.ncbi.nlm.nih.gov/source/11743__;!!Mih3wA!HSrsBsCuZsG7_Pyo42NlLz0omlaySnhN62kVOnugEkLkMLu8M2ivGWkxwOPdjSzYZE4bl5J_GPQj8RVXLh5gX7o$)

**Comment by kmartinez834** [comment]
Files from the links above are now in `downloads/snfg/current/`
For 2.6:
--> Add to files4nathan.csv so Nathan can provide an snfg xref file.  
--> Create download script and update download instruction documentation.

**Comment by edwardsnj** [comment]
@ubhuiyan It is not clear to me that we decided to add these monosaccharides to the glycan structure list for GlyGen. 

The second file, sia-table1.tsv, has its CID, SID values included in the Pubchem_substance_compound.csv file, so it appears redundant. Of the 225 CID values in Pubchem_substance_compound.csv file, 126 of them are in the GlyTouCan source set and therefore have GlyTouCan accessions associated with them. I have manually curated a similar SNFG mapping file, which may or may not agree with these values, but this is a nontrivial merge of information. 

Do we want multiple SID values associated with the CID (GlyTouCan + SNFG SIDs are distinct) of the mapped CIDs? 

I will add this to the agenda for Wednesday morning's meeting (7/10)

**Comment by edwardsnj** [comment]
@ubhuiyan It is not clear to me that we decided to add these monosaccharides to the glycan structure list for GlyGen. 

The second file, sia-table1.tsv, has its CID, SID values included in the Pubchem_substance_compound.csv file, so it appears redundant. Of the 225 CID values in Pubchem_substance_compound.csv file, 126 of them are in the GlyTouCan source set and therefore have GlyTouCan accessions associated with them. I have manually curated a similar SNFG mapping file, which may or may not agree with these values, but this is a nontrivial merge of information. 

Do we want multiple SID values associated with the CID (GlyTouCan + SNFG SIDs are distinct) of the mapped CIDs? 

I will add this to the agenda for Wednesday morning's meeting (7/10)
